blueprint/office-365.md

```diff
@@ -260,29 +265,32 @@ Add regional egress points | Not Configured | All traffic will egress centrally
 Enable split tunnelling connection for VPN users | Not Configured | All traffic will always traverse the VPN and egress through the internet gateway.
 
-Office 365 Connectivity Optimisation Design Decisions for an enhanced User Experience 
+Office 365 Connectivity Optimisation Design Decisions for an enhanced User Experience.
 
 Decision Point | Design Decision | Justification
 --- | --- | ---
-Workstation Connectivity | Direct connection to the internet | Provides the best performance for users
+Workstation Connectivity | Direct connection to the internet | Provides the best performance for users. 
 Local DNS resolution and Internet egress | Configured if required | DNS will be resolved to the gateway of their internet device.
 Add regional egress points | Configured if required | Regional Egress Points are not configured in this solution due to the workstations being directly connected to the internet.
 Bypass proxies and inspection devices | Configured | Proxies are not configured in this solution due to the workstations being directly connected to the internet.
-Enable split tunnelling connection for VPN users | Configured | Split tunnelling is configured in this solution to enable workstations to directly connect to Microsoft services. If a VPN solution is required Always on VPN should be integrated into the agency infrastructure
+Enable split tunnelling connection for VPN users | Configured | Split tunnelling is configured in this solution to enable workstations to directly connect to Microsoft services. If a VPN solution is required Always on VPN should be integrated into the agency infrastructure. 
 
 ### Exchange hybrid
 
-Exchange Online can be used standalone or integrated with an on-premises Exchange Server(s), extending the organisations messaging farm in a hybrid configuration. 
+This section is only relevant for agencies implementing a hybrid solution that leverages an on-premises Exchange Server(s).
 
-A Hybrid configuration provides administrators with added flexibility to transition users to the Cloud without isolating them from the on-premises resources. 
+Exchange Online can be used standalone (cloud only) or integrated with an on-premises Exchange Server(s) and Active Directory Domain Services, extending the organisations messaging farm in a hybrid configuration.
 
-This section is only relevant for agencies implementing a hybrid solution that leverages an on-premises Exchange Server(s)
+A Hybrid configuration provides administrators with added flexibility to transition users to the Cloud without isolating them from the on-premises resources. Hybrid with can also assist with transport routing for compliance reasons (e.g. Govlink) when Hybrid with "centralized mail transport" enabled. The [Edge Transport](https://docs.microsoft.com/en-us/exchange/edge-transport-servers) service may be deployed in scenarios where the agency does not wish to expose Hybrid mail servers directly to Exchange Online Protection.
```

Comment:
I think this sentence is incomplete. `Hybrid with` what?

```Hybrid with can also assist with transport routing for compliance reasons (e.g. Govlink) when Hybrid with "centralized mail transport" enabled.```

Reply:
have reworded to - 'when "centralized mail transport" is enabled.

Reply:
Sorry I should have been more clearer. I meant the first `Hybrid with` at the start of the sentence:

`Hybrid with can also assist with transport`

Reply:
@zedzedzed sorry missed that one. Updated to "A Hybrid configuration".